conflation/trace_fetching/mapillary.py

```diff
@@ -37,6 +33,26 @@
 # Name of the dir where we store sequence IDs pulled from Mapillary
 SEQUENCE_IDS_DIR_NAME = "seq_ids"
 
+# Mapillary API URLs
+COVERAGE_TILES_URL = (
+    "https://tiles.mapillary.com/maps/vtp/mly1_public/2/{}/{}/{}?access_token={}"
+)
+SEQUENCE_URL = "https://graph.mapillary.com/image_ids?fields=id&sequence_id={}&access_token={}"
+IMAGES_URL = "https://graph.mapillary.com/images?fields=captured_at,geometry&image_ids={}&access_token={}"
+
+# For all of the Mapillary calls, we need to rate limit them. The rate limit is 60k / min (last updated: 12/2021); we
+# give a small leeway to make sure we don't go over
+MAPILLARY_MAX_CALLS_PER_MINUTE = 59000
+MAPILLARY_PERIOD_MINUTE = 60
+
+
+def check_rate_limit_undecorated() -> None:
+    """
+    This method does nothing, but we will add rate-limiting decorators to it and use it as a buffer before any Mapillary
+    API calls.
```

Comment:
I decided to go with this pattern where I introduce one function that does nothing, rate limit it, and call it before any API calls. This way our rate limit can be global and our code can be a lot cleaner.